Formula/imagemagick.rb

```diff
@@ -129,11 +129,17 @@ def install
   end
 
   def caveats
-    s = <<-EOS.undent
+    base = <<-EOS.undent
+      rmagick 2.x does not work with ImageMagick 7, and some invocations of
+      `convert` may have changed syntax.
```

Comment:
Let's omit this; there's more than just `rmagick` and this isn't in keeping with other formulae updates.